src/components/Notification/Snackbar/Snackbar.tsx

```diff
@@ -75,17 +75,13 @@ const Snackbar: React.FC<SnackbarProps> = ({
           css={actionContainer()}
           data-testid={generateTestDataId('snackbar-secondary', dataTestId)}
         >
-          <Button type={'link'} isTransparent size="sm" onClick={secondaryCTA}>
-            {secondaryCTALabel}
-          </Button>
+          <Button onClick={secondaryCTA}>{secondaryCTALabel}</Button>
         </div>
         <div
           css={actionContainer()}
           data-testid={generateTestDataId('snackbar-primary', dataTestId)}
         >
-          <Button type={'link'} isTransparent size="sm" onClick={primaryCTA}>
-            {primaryCTALabel}
-          </Button>
+          <Button onClick={primaryCTA}>{primaryCTALabel}</Button>
```

Comment:
anything that has to do with `link` should have a @TODO for link component https://orfium.atlassian.net/jira/software/projects/NDS/boards/216?issueParent=30192&selectedIssue=NDS-536

Reply:
(will resolve in next PR)